source/mongorestore/mongorestore-examples.txt

```diff
@@ -60,7 +60,7 @@ To restore a specific collection, use :option:`--nsInclude
 <mongorestore --nsInclude>`, passing in the full namespace
 (``<database>.<collection>``) of the collection.
 
-For example, the following restores the collection named
+The following example restores the collection named
```

Comment:
Could we update https://www.mongodb.com/docs/manual/tutorial/backup-and-restore-tools/#std-label-backup-restore-oplogreplay and https://www.mongodb.com/docs/database-tools/mongorestore/#options as well as part of this? It seems like this page was part of what caused the user confusion about point in time restores